tests/spdx3/model/test_relationship.py

```diff
@@ -5,7 +5,7 @@
 
 import pytest
 
-from spdx_tools.spdx3.model.relationship import Relationship, RelationshipCompleteness, RelationshipType
+from spdx_tools.spdx3.model import Relationship, RelationshipCompleteness, RelationshipType
 
 
 @mock.patch("spdx_tools.spdx3.model.creation_information.CreationInformation", autospec=True)
```

Comment:
Nit: You could also adapt the paths in the mocks.

Reply:
Good point, added